test/integration/run.sh

```diff
@@ -25,8 +25,12 @@ export HELM_DATA_HOME="${helm_dir}/data"
 export HELM_HOME="${HELM_DATA_HOME}"
 export HELM_PLUGINS="${HELM_DATA_HOME}/plugins"
 export HELM_CONFIG_HOME="${helm_dir}/config"
-HELM_SECRETS_VERSION=3.5.0
 HELM_DIFF_VERSION=3.3.1
+if [ -z "${HELM_SECRETS_VERSION}" ]; then
+  HELM_SECRETS_VERSION="3.15.0"
+else
+  HELM_SECRETS_VERSION="${HELM_SECRETS_VERSION}"
+fi
```

Comment:
#!/bin/bash

b="default"

echo ${a-$b}
# default

a=""
echo ${a:-$b}  
# default

Reply:
@KqLLL

Reply:
```shell
DEFAULT_HELM_SECRETS_VERSION="3.15.0"
HELM_SECRETS_VERSION=${HELM_SECRETS_VERSION:-$DEFAULT_HELM_SECRETS_VERSION}
```
---
Is this feasible?

Reply:
@KqLLL I think it's fine. WDYT?

Reply:
```shell
HELM_SECRETS_DEFAULT_VERSION=3.15.0
HELM_SECRETS_VERSION="${HELM_SECRETS_VERSION:-$HELM_SECRETS_DEFAULT_VERSION}"
```
---

Which of the two variable names is better?

Reply:
all is fine.

Reply:
Okay. Thanks for the advice, I learned.